New log file opened at Thu Nov 17 16:36:25 COT 2016 by 3692@ISIS2503-0069
Taking next connection to: 59397
New client attempting to connect on port: 59398-59397
  Address = /172.24.42.64:59397
This client on port 59398 has uuid a264851e-b466-4b48-8fb0-697e5061b6a9 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-a264851e-b466-4b48-8fb0-697e5061b6a9
Taking next connection to: 59397
New client attempting to connect on port: 59399-59397
  Address = /172.24.42.64:59397
Handshake exception on socket: 59399
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59397
New client attempting to connect on port: 59400-59397
  Address = /172.24.42.64:59397
This client on port 59400 has uuid c3ef48fb-995b-4f1b-8391-4c97bd1d8976 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-a264851e-b466-4b48-8fb0-697e5061b6a9, activator-c3ef48fb-995b-4f1b-8391-4c97bd1d8976
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
New client attempting to connect on port: 59470-59397
  Address = /172.24.42.64:59397
Handshake exception on socket: 59470
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59397
New client attempting to connect on port: 59471-59397
  Address = /172.24.42.64:59397
This client on port 59471 has uuid d6148380-1584-429e-9bd9-e6ff42aeca90 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-a264851e-b466-4b48-8fb0-697e5061b6a9, play-fork-run-d6148380-1584-429e-9bd9-e6ff42aeca90
Taking next connection to: 59397
New client attempting to connect on port: 59541-59397
  Address = /172.24.42.64:59397
Handshake exception on socket: 59541
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59397
New client attempting to connect on port: 59542-59397
  Address = /172.24.42.64:59397
This client on port 59542 has uuid cfb7826e-b96f-472c-aa2d-0869909e255a configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-a264851e-b466-4b48-8fb0-697e5061b6a9, play-fork-run-cfb7826e-b96f-472c-aa2d-0869909e255a
Taking next connection to: 59397
New client attempting to connect on port: 59611-59397
  Address = /172.24.42.64:59397
Handshake exception on socket: 59611
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59397
New client attempting to connect on port: 59612-59397
  Address = /172.24.42.64:59397
This client on port 59612 has uuid c588e197-301c-4349-b735-59b7d6888a51 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-a264851e-b466-4b48-8fb0-697e5061b6a9, play-fork-run-c588e197-301c-4349-b735-59b7d6888a51
Taking next connection to: 59397
New client attempting to connect on port: 59650-59397
  Address = /172.24.42.64:59397
Handshake exception on socket: 59650
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59397
New client attempting to connect on port: 59651-59397
  Address = /172.24.42.64:59397
This client on port 59651 has uuid 3dae4b87-3fd5-4418-8f6c-f1c83c9e07ec configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-a264851e-b466-4b48-8fb0-697e5061b6a9, play-fork-run-3dae4b87-3fd5-4418-8f6c-f1c83c9e07ec
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
New client attempting to connect on port: 59813-59397
  Address = /172.24.42.64:59397
Handshake exception on socket: 59813
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59397
New client attempting to connect on port: 59814-59397
  Address = /172.24.42.64:59397
This client on port 59814 has uuid d8989f87-e396-4f2c-9166-613f69e7bb97 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-a264851e-b466-4b48-8fb0-697e5061b6a9, play-fork-run-d8989f87-e396-4f2c-9166-613f69e7bb97
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
New client attempting to connect on port: 59874-59397
  Address = /172.24.42.64:59397
Handshake exception on socket: 59874
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59397
New client attempting to connect on port: 59875-59397
  Address = /172.24.42.64:59397
This client on port 59875 has uuid 6f8f46b3-1eeb-4da6-ad8b-e0f79f24d9fd configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-a264851e-b466-4b48-8fb0-697e5061b6a9, play-fork-run-6f8f46b3-1eeb-4da6-ad8b-e0f79f24d9fd
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59397
